board_client/animations/boing_ball.py

```
@@ -1,12 +1,18 @@
 import uasyncio
 import math
-import random
+import time
```

Ruff (0.11.9)

`time` imported but unused

Remove unused import: `time`

(F401)